packages/core/src/common/_color-aliases.scss

```diff
@@ -25,6 +25,10 @@ $pt-app-elevated-background-color: $light-gray4 !default;
 $pt-dark-app-elevated-background-color: $dark-gray3 !default;
 
 $pt-outline-color: rgba($blue3, 0.6) !default;
```

Comment:
Not changing this color because of the comment at the top of this file stating that is generally considered a breaking change. Perhaps we want to make an exception here...